cascade.py

```
@@ -351,106 +794,17 @@ def queque_worker(self):
-        if self.patience_counter > 2:
-            self.PATIENCE_FLAG = True
-        if self.face_counter > 1:
+        if self.patience_counter > 2 or self.face_counter > 1:
             self.PATIENCE_FLAG = True
 
     def single_debug(self):
         start_time = time.time()
-        target_img_name = 'dummy_img.jpg'
-        target_img = cv2.imread(os.path.join(cat_cam_py, 'CatPreyAnalyzer/readme_images/lenna_casc_Node1_001557_02_2020_05_24_09-49-35.jpg'))
+        target_img_name = "dummy_img.jpg"
+        target_img = cv2.imread(os.path.join(cat_cam_py, "CatPreyAnalyzer/readme_images/lenna_casc_Node1_001557_02_2020_05_24_09-49-35.jpg"))
```

Ruff (0.11.9)

Undefined name `cat_cam_py`

(F821)